prisma/schema.prisma

```diff
@@ -10,8 +10,39 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Employee{
-  employeeId Int @id @default(autoincrement())
-  name String
-  salary Int @default(100000)
+model Employee {
+  employeeId   Int        @id @default(autoincrement())
+  name         String
+  salary       Int        @default(100000)
+  designation  Designation @default(Junior)
+  department   Department @relation(fields: [departmentId], references: [departmentId])
+  departmentId Int
+  projects ProjectAssignment[]
+}
+
+model Department {
+  departmentId    Int        @id @default(autoincrement())
+  departmentName String
+  Employee       Employee[]
```

Comment:
should be named employees

Reply:
I have done the changes. Its in local. I'll push it.

Reply:
This has been fixed in the lastest commit